public/js/components/Frames.js

```diff
@@ -8,22 +9,24 @@ const actions = require("../actions");
 const { endTruncateStr } = require("../utils/utils");
 const { getFilename } = require("../utils/source");
 const { getFrames, getSelectedFrame, getSource } = require("../selectors");
+const { List } = require("immutable");
+const classNames = require("classnames");
 
 if (typeof window == "object") {
   require("./Frames.css");
 }
 
 const NUM_FRAMES_SHOWN = 7;
 
-function renderFrameTitle(frame) {
-  return div({ className: "title" }, endTruncateStr(frame.displayName, 40));
+function renderFrameTitle({ displayName }: Object) {
```

Comment:
Yea, you can define the type above the function and then use it. Perhaps that's nice... Or just add a line break... Also, I'm not convinced we need it. Your call.